meteor.module

```diff
@@ -25,10 +25,35 @@ use Drupal\user\UserInterface;
  *
  * @see \Drupal\meteor\IdentityListener
  */
-function _meteor_notify($event, AccountInterface $account = NULL, $delay = 0) {
+function _meteor_notify_user_event($event, AccountInterface $account = NULL, $delay = 0) {
```

Comment:
I prefer to have functions in alphabetical order in files: this one goes below `_meteor_notify`.